src/services/legal-services.ts

```diff
@@ -41,11 +41,20 @@ export default class LegalServices {
   /**
    * Fetches filings list.
    * @param businessId the business identifier (aka entity inc no)
-   * @returns the axios response
+   * @returns the filings list
    */
-  static async fetchFilings (businessId: string): Promise<AxiosResponse> {
+  static async fetchFilings (businessId: string): Promise<ApiFilingIF[]> {
     const url = `businesses/${businessId}/filings`
     return axios.get(url)
+      .then(response => {
+        const filings = response?.data?.filings as ApiFilingIF[]
+        if (!filings) {
+          // eslint-disable-next-line no-console
+          console.log('fetchFilings() error - invalid response =', response)
+          throw new Error('Invalid filings list')
+        }
+        return filings
+      })
```

Comment:
This now returns a list of API Filing objects.

When we refactor the Todo List, we should do the same thing for tasks.

Reply:
Instead of throwing an error, what do you think about having `fetchFilings()` always return a promised instead.  This approach would then allow downstream logic to gracefully handle the error in the most appropriate way.  When we throw an error, doesn't the browser just display an error in the console log?  The regular, non-technical user wouldn't know what had happened -- right?

Reply:
This error handling both logs an error to the console (with the subject response) and throws an error, which is super-easy to handle in the caller. For example:
```
const filings = await fetchFilings(123)
   .then (/* do something with the return value */)
   .catch (/* do something with the error */)
```
or
```
try {
   const filings = await fetchFilings(123)
   /* do something with the return value */
} catch (error) {
   /* do something with the error */
}
```